docs/getting-started/prerequisites.md

```diff
@@ -0,0 +1,237 @@
+# Setting up the prerequisites
+
+This guideline includes the steps to install the prerequisites with specified versions for the Carbyne Stack. All installation procedures are done with the Terminal application of Ubuntu. **Please open the Terminal application before you proceed.**
+
```

Comment:
Please add the following disclaimer here:

```
!!! warning
    Carbyne Stack has been tested using the **exact versions** of the tools
    specified below. Deviating from this _battle tested_ configuration may
    create all kinds of issues.
```

Reply:
@strieflin Thanks for all the suggestions. I addressed all the suggestions below in the new commit.

The disclaimer is now added.